### PR TITLE
add hero styling and background image

### PR DESCRIPTION
**Pull request by wrightmartin**

### CODE REVIEW

File: lib/views/general/_frontpage_intro_sentence.html.erb [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<h2>
-  Your <strong>Right to Know</strong>
-</h2>
-<p>Every citizen has the right to access information held by public bodies. <strong>By law, they have to respond</strong>. <a href="<%= help_about_url %>">Find out more about Freedom of Information.</a></p>
+<h1 class="intro__title"><%= _("Your right to know") %></h1>
+<p class="intro__description"><%= _("Every citizen has the right to access information held by public bodies. By law, they have to respond") %></p>
+
```
Comment (garethrees) [inline lib/views/general/_frontpage_intro_sentence.html.erb]: I don't think alavetelitheme should override this template, as this isn't adding anything special. We should use the core version. I actually prefer this copy, so happy to port it to core.

Comment (garethrees) [inline lib/views/general/_frontpage_intro_sentence.html.erb]: I'd make a minor change:

```diff
- Every citizen has the right to access information held by public bodies
+ Every citizen has the right to access information held by public authorities
```

File: assets/stylesheets/responsive/custom.scss [hunk 2]
```diff
@@ -582,3 +582,63 @@ dt {
...
+      background-color: $color_blue;
+      border-color: darken($color_blue, 15%);
+      transition: color 300ms ease-out;
+    }
+  }
+}
+
+.intro__title {
+  color: $color_white;
+  text-shadow: 0 1px 2px rgba(0,0,0,0.3);
```
Comment (garethrees) [inline assets/stylesheets/responsive/custom.scss]: `transparentize $color_black`

**Comment by garethrees** [comment]
Love this PR so much. Really brings everything together. two minor issues to address but looking excellent :+1:

**Comment by wrightmartin** [comment]
Thanks gareth, I've made those changes and pushed them

**Pull request merged**